### PR TITLE
Migrate to Vue CLI 3 for UI kit development mode

### PR DESCRIPTION
**Pull request by MrLyfing**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -115,7 +111,6 @@ methods: {
 components: {
   'hf-action-button': ActionButton
```
Comment (ivdekov) [inline README.md]: Do we want to remove `hf` from these example snippets?

Comment (MrLyfing) [reply README.md]: I'll update this.

File: package.json [hunk 2]
```diff
@@ -5,10 +5,16 @@
   "private": true,
```
Comment (ivdekov) [inline package.json]: Does the version number have to be updated?

File: .eslintrc.js [hunk 3]
```diff
@@ -1,44 +1,17 @@
 module.exports = {
   root: true,
-  parserOptions: {
-    parser: 'babel-eslint'
-  },
   env: {
-    browser: true
+    node: true
```
Comment (syn-zeta) [inline .eslintrc.js]: No node support.

File: components/ActionButton.vue [hunk 4]
```diff
@@ -45,26 +45,26 @@ export default {
       type: String,
       default: ''
     },
-    disabled: Boolean,
+    disabled: Boolean
   },
 
-  data() {
+  data () {
```
Comment (syn-zeta) [inline components/ActionButton.vue]: Can you please update the lint config to not have spaces before parens here?

File: kit/pages/main.js [hunk 5]
```diff
@@ -187,34 +187,34 @@ export default {
       },
       assignOptions: [{
         label: 'Signed In',
-        options: [{user: {first_name: 'John', user_id: 1, 'last_name': 'Doe'}},
-          {user: {first_name: 'Tom', user_id: 2, 'last_name': 'Sawyer'}},
-          {user: {first_name: 'Micky', user_id: 3, 'last_name': 'Mouse'}},
-          {user: {first_name: 'Rodger', user_id: 4, 'last_name': 'Federer'}}]
+        options: [{ user: { first_name: 'John', user_id: 1, 'last_name': 'Doe' } },
```
Comment (syn-zeta) [inline kit/pages/main.js]: Do you have any preference here? I prefer the Pythonic way, to the left, in this case.

**Pull request merged**